### PR TITLE
HAWQ-1580. Change PXF port number to 5888 for GPDB

### PR DESCRIPTION
**Pull request by lavjain**

### CODE REVIEW

File: pxf/pxf-service/src/scripts/pxf-service [hunk 1]
```diff
@@ -21,25 +21,25 @@
...
-if [ ! -f $env_script ]; then
+if [ ! -f ${env_script} ]; then
 	echo WARNING: failed to find $env_script
 else
-	source $env_script
+	source ${env_script}
 fi
 
 pxf_user=${PXF_USER}
 instance_port=${PXF_PORT:-51200}
```
Comment (shivzone) [inline pxf/pxf-service/src/scripts/pxf-service]: Just confirming, do we want the service script to default to 51200 if PXF_PORT is not set ?

**Comment by shivzone** [comment]
There are some references to port 51200 in some of our rest resource classes and also in the pxf-jdbc markdown file. Please update them as well.

File: pxf/pxf-service/src/scripts/pxf-service [hunk 2]
```diff
@@ -185,91 +185,74 @@ function doInit()
...
+    popd > /dev/null
 
-    context_file=$instance_root/$instance_name/webapps/pxf/META-INF/context.xml
-    cat $context_file | \
-    sed  -e "s:classpathFiles=\"[a-zA-Z0-9\/\;.-]*\":classpathFiles=\"$PXF_HOME\/conf\/pxf-private.classpath\":" \
-    -e "s:secondaryClasspathFiles=\"[a-zA-Z0-9\/\;.-]*\":secondaryClasspathFiles=\"$PXF_HOME\/conf\/pxf-public.classpath\":" > context.xml.tmp
-    mv context.xml.tmp $context_file
+    context_file=${instance_root}/${instance_name}/webapps/pxf/META-INF/context.xml
+    sed -i -e "s:classpathFiles=\"[a-zA-Z0-9\/\;.-]*\":classpathFiles=\"${PXF_HOME}\/conf\/pxf-private.classpath\":" \
+           -e "s:secondaryClasspathFiles=\"[a-zA-Z0-9\/\;.-]*\":secondaryClasspathFiles=\"${PXF_HOME}\/conf\/pxf-public.classpath\":" ${context_file}
```
Comment (denalex) [inline pxf/pxf-service/src/scripts/pxf-service]: nice, thanks for cleaning this up

**Comment by denalex** [comment]
Will we also update HAWQ with the new default value ?

**Comment by lavjain** [comment]
https://issues.apache.org/jira/projects/HAWQ/issues/HAWQ-1580

**Pull request closed without merging**